common/src/main/java/org/apache/nemo/common/dag/DAGBuilder.java

```diff
@@ -169,6 +176,94 @@ public DAGBuilder(final DAG<V, E> dag) {
     return this;
   }
 
+  /**
+   * Use this method when there are no edges to be replaced.
+   * @param edgeToReference edge to reference.
+   * @param edgeToInsert  edge to insert.
+   * @return  itself.
+   */
+  public DAGBuilder<V, E> connectSplitterVertexWithoutReplacing(final E edgeToReference, final E edgeToInsert) {
+    final V src = edgeToInsert.getSrc();
+    final V dst = edgeToInsert.getDst();
+    LOG.error("edgeToReference: {} {} {}", edgeToReference.getId(), edgeToReference.getSrc().getId(),
```

Comment:
Please use the logs accordingly!

Reply:
Thanks for your mention! I must have forgotten to erase the LOG.

Erased it. Thanks!

Reply:
I must have forgotten to erase the logs. Thanks!